tests/e2e/files_test.go

```diff
@@ -322,438 +317,3 @@ func TestFileEncryptionDecryption(t *testing.T) {
 	assert.Equal(fileTextIn, fileTextDec)
 
 }
-
```

Comment:
Q: love when there is more deleted than added lines but what was that lines? They seem to be some weirdly written tests - Am I right? 
Do you think that they're no longer required or the added ones satisfied removed test cases?

Reply:
None of those tests have been running, so I just removed them